documentation/modules/creating-storage-mapping.adoc

```diff
@@ -25,9 +25,10 @@ You can create a storage mapping by using the {ocp} web console to map source di
 
 . Map source disk storages to target storage classes as follows:
 
-.. If your source provider is VMware, select a *Source datastore* and a *Target storage class*.
+.. If your source provider is VMware vSphere, select a *Source datastore* and a *Target storage class*.
 .. If your source provider is {rhv-full}, select a *Source storage domain* and a *Target storage class*.
 .. If your source provider is {osp}, select a *Source volume type* and a *Target storage class*.
+.. If your source provider is a set of one or more OVA files, select a *Source* and a *Target storage class*.
```

Comment:
update regarding the storage mapping, so since OVA doesn't really has storage mapped we added a dummy storage for each provider will be in the following format.
`Dummy storage for <provider-name>`, I'm not sure how we should call it here...

Reply:
+we will need to update this part for the CLI as well, need to be mention to use the name filed (as specified above) for the yaml file.

Reply:
@qiyuann When you tested OCP-OCP, what did you see in Storage Mapping? It looked like *Source* and *Target storage class* when I used the providers you created, but I could be wrong.
@bkhizgiy -- my goal here is to describe what the user sees in the UI.

Reply:
@RichardHoch right but what @bkhizgiy meant is that there was a recent change that added the 'Dummy storage' so if this documentation is based on observation based on an older build, this needs to change

Reply:
@ahadas Understood ... and that's what I am seeing now:
![OVA stoarge mapping](https://github.com/kubev2v/forklift-documentation/assets/60698649/a86e35c0-7f54-4ced-a4db-8f52fa7b269a)

Reply:
yeah, so maybe in this case we can say something like:
If your source provider is a set of one or more OVA files, select a *Target storage class* for the 'Dummy storage' that applies to all virtual disks within the OVA files.